pkg/machinery/resources/secrets/trustd.go

```diff
@@ -27,8 +27,8 @@ type Trustd = typed.Resource[TrustdCertsSpec, TrustdExtension]
 //
 //gotagsrewrite:gen
 type TrustdCertsSpec struct {
-	CA     *x509.PEMEncodedCertificateAndKey `yaml:"ca" protobuf:"1"` // only cert is passed, without key
```

Comment:
`ca` field is removed?

Reply:
it was actually "CAs to trust" field, and now it's more explicit